modules/src/ics02_client/handler.rs

```diff
@@ -6,7 +8,6 @@ use crate::ics24_host::identifier::ClientId;
 use crate::ics02_client::context::ClientReader;
 use crate::ics02_client::handler::create_client::CreateClientResult;
```

Comment:
It would be nice for this to read:
```suggestion
use crate::ics02_client::handler::create_client::Result;
```